assignment/src/main/java/nerdschool/bar/Pub.java

```diff
@@ -1,69 +1,7 @@
 package nerdschool.bar;
 
+// A copy for commenting code
+	
 public class Pub {
 
-
-    public static final String ONE_BEER = "hansa";
-    public static final String ONE_CIDER = "grans";
-    public static final String A_PROPER_CIDER = "strongbow";
-    public static final String GT = "gt";
-    public static final String BACARDI_SPECIAL = "bacardi_special";
-
```

Comment:
Create a Pub() constructor to initialize the menu. The menu can be a Map<String, Drink>. With it, we can specify existing drinks and manage the exception when an unknown drink is specified